#std_msgs/String location_name
string location_name
---
#std_msgs/Bool  success
bool success
